src/client/dfuse/dfuse_main.c

```diff
@@ -99,6 +99,21 @@ dfuse_launch_fuse(struct dfuse_info *dfuse_info,
 	return false;
 }
 
+static void
+show_help(char *name)
+{
+	printf("usage: %s -m <mountpoint>\n"
+		"\n"
+		"	-m --mountpoint <path>	Mount point to use\n"
+		"	-s --svcl <svcl>	DAOS svcl\n"
+		"	-p --pool <uuid>	Connect to pool\n"
+		"	-c --container <uuid>	Connect to container\n"
+		"	-g --group <group name>	CaRT group\n"
```

Comment:
I can change that easily enough.

Reply:
@kccain does this look consistent with the daos command now?

Reply:
We can also consider the daos_shell pool create/destroy usage. And this community wiki page describes the syntax we are aiming for (although of course questions/adjustments will arise in development) https://wiki.hpdd.intel.com/display/DC/DAOS+Tools

I’m pretty sure daos command does not implement the short options. The struct option has characters that identify each option of course, but I don't think the command actually accepts them. So this may be (in a way) less of a concern for collisions with other commands.

DAOS system name  daos has--sys-name=, --sys= shorter form due to getopt_long() and –sys= being a unique substring
daos_shell pool: uses --sys= (and uses -S as a short option)
so here in this command: perhaps -S could be considered as a short option (realize this conflicts with single threaded option)

Service replica list daos –svc= and pool –pool=<uuid> seem to agree with the latest commit here.

Container:  daos --cont=<uuid>
So here in this command: if --container= is used, perhaps getopt_long will give us --cont= for free? Or this command could just accept  --cont= but not --container=

(I don't think this impacts the latest commit here that does not use -g, but just for some context) one of the areas that came up while discussing daos and the (old/legacy) dmg and new daos_shell is the term "group". Definitely want to use system name where that makes sense. Because with daos_shell pool create it uses --group= and -g shorthand to refer to the group that owns the pool.

Reply:
> We can also consider the daos_shell pool create/destroy usage. And this community wiki page describes the syntax we are aiming for (although of course questions/adjustments will arise in development) https://wiki.hpdd.intel.com/display/DC/DAOS+Tools

Perfect, that's what I was looking for.

> I’m pretty sure daos command does not implement the short options. The struct option has characters that identify each option of course, but I don't think the command actually accepts them. So this may be (in a way) less of a concern for collisions with other commands.

Ah, I see.  The third option to getopt_long() is "" in daos.c which means presumably it doesn't.

The implication from this is that I should remove at least the -p -c -s and -G short options from dfuse.